### DISCUSSION TITLE
URLs in user profiles do not work or only work inconsistently

### DISCUSSION THREAD

**Discussion by kydecker** [community/community#78919] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Custom URLs (for portfolios, personal websites, etc.) in user profiles on the GitHub mobile app for iOS are broken for me or only work rarely. Tapping the URL does nothing and does not open the browser. There is no way to select and copy the text of the URL as a workaround.

**Comment by stevepopovich** [comment]
Hi @evadecker thanks for writing in and I am sorry this is happening. I am having trouble reproducing this, do you have a profile this is happening in?

**Reply by kydecker** [reply]
Sure: this profile is one example: https://github.com/jonhargreaves

https://github.com/community/community/assets/4117920/01b65106-cc7b-42f7-a2e3-325e6d728de3

This video shows my experience tapping "www.coffeeclub.net" with no response.

**Reply by stevepopovich** [reply, marked as answer]
amazing. This is great. I can reproduce now. I will file a bug with the team and they will drop back here when the issue is fixed. Thanks for writing in!

**Reply by kydecker** [reply]
Thank you @stevepopovich!

**Reply by kydecker** [reply]
Another example of a profile containing a URL which doesn't respond: https://github.com/seanehalpin

**Comment by 26e43ee** [comment]
How reslove this problem
